app_check/src/desktop/app_check_desktop.cc

```diff
@@ -126,6 +130,45 @@ void AppCheckInternal::RemoveAppCheckListener(AppCheckListener* listener) {
   }
 }
 
+int g_app_check_registry_count = 0;
```

Comment:
make this a static int since it is only relevant to this file

Reply:
Done